backend/prepare.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This could be a sh script, but this script will really only run once when building and bash-specific feathers may be added in the future.
The point is that bash is the provided environment in this base image, and I cannot find a reason not to use it.